pkg/controller/servicemesh/webhookca/wrappers.go

```diff
@@ -18,17 +21,42 @@ type webhookGetter interface {
 type webhookWrapper interface {
 	MetaObject() metav1.Object
 	Object() runtime.Object
-	ClientConfigs() []*v1beta1.WebhookClientConfig
+	ClientConfigs() []*v1.WebhookClientConfig
 	Copy() webhookWrapper
 	NamespacedName() types.NamespacedName
+	UpdateCABundle(ctx context.Context, cl client.Client, caBundle []byte) error
 }
 
 func toWebhookWrapper(obj runtime.Object) (webhookWrapper, error) {
 	switch wh := obj.(type) {
-	case *v1beta1.ValidatingWebhookConfiguration:
+	case *v1.ValidatingWebhookConfiguration:
 		return &validatingWebhookWrapper{ValidatingWebhookConfiguration: wh}, nil
-	case *v1beta1.MutatingWebhookConfiguration:
+	case *v1.MutatingWebhookConfiguration:
 		return &mutatingWebhookWrapper{MutatingWebhookConfiguration: wh}, nil
+	case *apixv1.CustomResourceDefinition:
+		return &conversionWebhookWrapper{CustomResourceDefinition: wh}, nil
 	}
-	return nil, fmt.Errorf("Object is not a MutatingWebhookConfiguration or ValidatingWebhookConfiguration")
+	return nil, fmt.Errorf("Object is not a MutatingWebhookConfiguration or ValidatingWebhookConfiguration: %T", obj)
```

Comment:
...and it's also not a CRD - I guess now we should use the `object not in [list]` style

Reply:
done